November newsletter blog (#269)
ddev/ddev.com: src/content/blog/ddev-nov-2024-newsletter.md

---
title: "DDEV November 2024 Newsletter"
pubDate: 2024-11-01
#modifiedDate: 2024-09-06
summary: DDEV November 2024 Newsletter
author: Randy Fay
featureImage:
  src: /img/blog/2024/11/palisade-november-bookcliffs.jpg
  alt: Palisade, Colorado, USA, November 1, 2024
categories:
  - Community
---

**Happy November from DDEV**! It's a beautiful fall here, but things are cooling down. We live in an agricultural area, so the first freeze is always an event.

> **It's Maintainer [Stas Zhuk]'s first anniversary as a full DDEV maintainer!**. I asked him to share what he's most proud of this year. I know how amazingly grateful I am to work alongside him and to such an amazingly capable (and fully privileged) co-maintainer:

Reviewer: ```suggestion
**It's Maintainer [Stas Zhuk]'s first anniversary as a full DDEV maintainer!**. I asked him to share what he's most proud of this year. I know how amazingly grateful I am to work alongside him and to such an amazingly capable co-maintainer who has full privileges for the DDEV repository.
```

the suggestion is not perfect at all wording wise. but i just wanted to add some context to "fully privileged". cuz without the context i was uncertain what privileged was actually referring to and what about. about my suggestion, i was uncertain about the scope. is he fully privileged for the ddev repo or the org in general. that might need some readjustment same as the wording in general has room to improve.

Author: ```suggestion
**It's Maintainer [Stas Zhuk]'s first anniversary as a full DDEV maintainer!**. I asked him to share what he's most proud of this year. I know how amazingly grateful I am to work alongside him and to such an amazingly capable co-maintainer who has full privileges on all things DDEV.
```

> When I started using DDEV, I already knew a bit about Docker, but I've learned a lot more by working on DDEV's Docker networks. I've made a lot of PRs to improve this.

> I also had my first experience with Docker multi-stage builds in DDEV. We use it to compile Xdebug for some PHP versions, and it was fun to learn.

> The `ddev launch :port` feature has been really useful. It's hard to imagine that DDEV didn't have it before.

> I recently added automatic timezone detection in the web container. It seems like a small thing, but it has a big impact on simplicity and is very convenient for developers.

> Finally, the upcoming `ddev cd` feature is something I’m excited about. I used to have a custom Bash script for this, but now it will be built into DDEV, with autocomplete.

**Advisory Group Meeting Wednesday**: All are invited to the DDEV Advisory Group meeting Wednesday, November 6, 9am US MT, 5PM CET. [Meetup link](https://www.meetup.com/ddev-events/events/303197392/?eventOrigin=group_events_list) or [Zoom link](https://us02web.zoom.us/j/7315692237?pwd=RHR6NUkwb0g5WXIzS2NOcXRucCthZz09&omn=89153105842), passcode "12345" if you need it.

> **Contributor Trainings From October (Recordings and Blogs)**

Reviewer 2: ```suggestion
**Contributor Trainings From October (Recordings and Blogs):**
```

> * [Supporting people using `ddev debug test` and other tools](ddev-debug-test-contributor-training.md)
> * [Using Tmate to Debug GitHub Workflows**](tmate-github-actions-contributor-training.md)

Reviewer 2: ```suggestion
* **[Supporting people using `ddev debug test` and other tools](ddev-debug-test-contributor-training.md)**
* **[Using Tmate to Debug GitHub Workflows](tmate-github-actions-contributor-training.md)**
```

**DDEV Notes from around the web:**

> - **[DDEV Release v1.23.5 Announcement](https://ddev.com/blog/release-v1235-auto-port-assignment). Lots of goodies in this latest release, including automatic port assignment and automatic time zone detection.

Reviewer 2: ```suggestion
- **[DDEV Release v1.23.5 Announcement](https://ddev.com/blog/release-v1235-auto-port-assignment).** Lots of goodies in this latest release, including automatic port assignment and automatic time zone detection.
```

> - **[Mauricio Dinarte (@dinarcon)'s DrupalCon presentation on DDEV](https://www.youtube.com/watch?v=nPJC7BbiGNw&list=PLpeDXSh4nHjQOfQV-BUgoxHXlr4tHlhPO&index=16&pp=iAQB)**

Reviewer: ```suggestion
- **[Mauricio Dinarte's (@dinarcon) DrupalCon presentation on DDEV](https://www.youtube.com/watch?v=nPJC7BbiGNw&list=PLpeDXSh4nHjQOfQV-BUgoxHXlr4tHlhPO&index=16&pp=iAQB)**
```
